TreeTracker/UI/Views/AddTree/AddTreeViewController.swift

```diff
@@ -100,6 +118,17 @@ private extension AddTreeViewController {
     }
 }
 
+// MARK: - Class Methods
+func animatedImages(for name: String) -> [UIImage] {
```

Comment:
See what you're doing here which is kinda clever but i would probably just keep it simple and set the images using the swiftgen generated assets class, that way if they're ever missing the app won't complile. I'd probably just do it in the didSet and remove this function all together.
```
searchGPSSignalImageView.animationImages = [
    Asset.Assets.GpsSearchAnimation.gpsLoad0.image,
    Asset.Assets.GpsSearchAnimation.gpsLoad1.image,
    Asset.Assets.GpsSearchAnimation.gpsLoad2.image
]
```
or maybe 
```
searchGPSSignalImageView.animationImages = [
    Asset.Assets.GpsSearchAnimation.gpsLoad0,
    Asset.Assets.GpsSearchAnimation.gpsLoad1,
    Asset.Assets.GpsSearchAnimation.gpsLoad2
].map({$0.image})
```
If you want to be more _'swifty'_